game-core/src/test/java/games/strategy/engine/lobby/client/login/LobbyServerPropertiesFetcherTest.java

```diff
@@ -136,13 +126,13 @@ void shouldReturnEmptyWhenHostNotSetAndPortNotSet() {
     }
 
     @Test
-    void shouldReturnEmptyWhenHostSetAndPortNotSet() {
-      givenTestLobbyHostIsSet();
-      givenTestLobbyPortIsNotSet();
+    void shouldReturnDefaultPortWhenHostIsSetAndPortNotSet() {
+      givenTestLobbyHostIsSetTo("foo");
+      // givenTestLobbyPortIsNotSet();
```

Comment:
I think you missed to remove this comment

Reply:
cbd223084c1928bcad78e0e6bbbf8b59e9e113b8